.conan/profiles/config/min-size-rel

```diff
@@ -1,2 +1,5 @@
 [settings]
-build_type=MinSizeRel
\ No newline at end of file
+build_type=MinSizeRel
+mold*:build_type=Release
+mold*:compiler.libcxx=libstdc++11
```

Comment:
Clang tries to build mold with `libc++` which produces a warning saying it must be built with `libstdc++11` so we override this also in the setting. Duplicated for now, but should also only be one instance shared amongst settings.

Reply:
Perfect, the setting `mold*:compiler.libcxx=libstdc++11` has been added to the common configuration file.